Formula/neomutt.rb

```diff
@@ -0,0 +1,72 @@
+class Neomutt < Formula
+  desc "Teaching an Old Dog New Tricks"
```

Comment:
This needs to be a description not a tag line.

Reply:
@jindraj How about: "Email reader with support for Notmuch (fast indexing), News (NNTP) and much more"